types/react-smartbanner/package.json

```diff
@@ -6,7 +6,7 @@
         "https://github.com/patw0929/react-smartbanner#readme"
     ],
     "dependencies": {
-        "@types/react": "^18.3.9999"
+        "@types/react": "^18.3.11"
```

Comment:
I'm not sure why this is pinned at all. Why is this not `^18`?

Reply:
You're right, but technically it doesn't make it a difference, it's more a semantics thing